docs/pages/accordion-menu.md

```diff
@@ -76,7 +76,7 @@ You need to add the class `accordion-menu` as well as the data attribute `data-s
 ```html
 <ul class="vertical menu accordion-menu" data-accordion-menu data-submenu-toggle="true">
   <li>
-    <a href="http://foundation.zurb.com/">Zurb Foundation</a>
+    <a href="https://www.get.foundation/">Zurb Foundation</a>
```

Comment:
Same like the others. Shorter === better.

Reply:
Shorter is not always better. I run my business off a non-standard TLD. I understand your view. I dislike www as well. 

I guess I could set it up to redirect www so that we can use it when we need it.

Reply:
> I guess I could set it up to redirect www so that we can use it when we need it.

That would be great.